### PR TITLE
Fix vault import/export

### PR DESCRIPTION
**Pull request by aminsato**

### CODE REVIEW

**Comment by johnnyluo** [comment]
This PR is quite weird , a lot changes made by @YGolybev have been merged to main show up in your PR again 

Could you please merge main to your branch?

**Comment by Ehsan-saradar** [comment]
> This PR is quite weird , a lot changes made by @YGolybev have been merged to main show up in your PR again
> 
> Could you please merge main to your branch?

Fixed. Check again, please

**Comment by aminsato** [comment]
> This PR is quite weird , a lot changes made by @YGolybev have been merged to main show up in your PR again
> 
> Could you please merge main to your branch?

Fixed, please  check again

File: app/src/main/java/com/vultisig/wallet/data/on_board/db/VaultDB.kt [hunk 1]
```diff
@@ -16,14 +24,22 @@ class VaultDB(context: Context) {
         vaultsFolder.mkdirs()
     }
 
+    // Delete operation
+    fun delete(vaultName: String) {
+        val file = vaultsFolder.resolve("${vaultName}${FILE_POSTFIX}")
+        if (file.exists()) {
+            file.delete()
+        }
+    }
```
Comment (YGolybev) [inline app/src/main/java/com/vultisig/wallet/data/on_board/db/VaultDB.kt:33]: I can't fined usage of this method, consider removing it

**Review by YGolybev — APPROVED** [review]

**Pull request merged**